pegasus_ba.ts

```diff
@@ -0,0 +1,511 @@
+        <source>y axis</source>
+        <translation>y osa</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="348"/>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="545"/>
+        <source>press</source>
+        <translation>pritisni</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="367"/>
+        <source>right back</source>
+        <translation>desno dugme iza</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="415"/>
+        <source>abxy</source>
+        <translation>ABXY</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="498"/>
+        <source>right stick</source>
+        <translation>desni stik</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="570"/>
+        <source>center</source>
+        <translation>centar</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="579"/>
+        <source>select</source>
+        <translation>selekt</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="599"/>
+        <source>guide</source>
+        <translation>vodič</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="618"/>
+        <source>start</source>
+        <translation>start</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="714"/>
+        <source>hold down to quit</source>
+        <translation>drži dole da izađeš</translation>
+    </message>
+</context>
+<context>
+    <name>GenericOkCancelDialog</name>
+    <message>
+        <location filename="../src/frontend/dialogs/GenericOkCancelDialog.qml" line="160"/>
+        <source>Ok</source>
+        <translation>OK</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/dialogs/GenericOkCancelDialog.qml" line="196"/>
+        <source>Cancel</source>
+        <translation>Otkaži</translation>
+    </message>
+</context>
+<context>
+    <name>GenericOkDialog</name>
+    <message>
+        <location filename="../src/frontend/dialogs/GenericOkDialog.qml" line="136"/>
+        <source>Ok</source>
+        <translation>OK</translation>
+    </message>
+</context>
+<context>
+    <name>HelpScreen</name>
+    <message>
+        <location filename="../src/frontend/menu/HelpScreen.qml" line="66"/>
+        <source>About</source>
+        <translation>O Softveru</translation>
```

Comment:
Small "s" might look better on this screen, or is this style preferred in this language?

Reply:
Nobody cares that much, but if you think a small letter is better in this case, then I'll change it immediately.

If anyone else wants to change it to a big letter later, let them.